### ISSUE TITLE
Laptop keyboard function comes and goes in Gecko rolling MATE edition

### ISSUE THREAD

**Issue by esteelpaz** [geckolinux/geckolinux-project#232]
@geckolinux 

As stated in the subject line I can't seem to get off the banana peel with SUSE flavors lately . . . the "wiping of grub menu" problem in TW & Gecko still remains on my two Macs . . . and now on my newer '20 Sys76 linux laptop I've had an issue where the keyboard function seems to be "intermittent" in Gecko . . . .

I have a multi-boot situation on this machine, so I'm now in Pop/ubuntu typing this post, but when I booted into Gecko the mouse clicking works, but trying to type with the keyboard sometimes it works and sometimes not???  In the first occurrence it didn't work, I think I rebooted, went back into Gecko . . . keyboard worked, then I ran another zypper dup. . . and rather than wiping grub, it seems to wipe the keyboard?? . . . .

So, the problem repeated beyond a single incident . . . .  Anything to test out or provide log files on?

**Comment by geckolinux** [comment]
I've read about weird things happening with the firmware when switching between Windows and Linux with just a reboot due to values remaining set in the memory. Maybe try completely powering off and then booting GeckoLinux.

**Comment by esteelpaz** [comment]
@geckolinux 

Thanks for the reply, in this case on the Sys76 I just have 3 linux distros . . . and lately I have been restarting between them because I have been trying to troubleshoot problems in the Pop/ubuntu install.  But, usually I do completely shut down at the end of the day's work/play with it.

I thought it might have been because at home I use it with an ext keyboard plugged in, but at work I just have an ext mouse plugged in . . . problem seemed to show up at work.

Yesterday I booted Gecko and ran a zypper, with the ext keyboard unplugged . . . then plugged it in . . . problem didn't seem to show up at all . . . .  I'll "monitor" the situation for a bit . . . possibly taking it to work tomorrow some 24 hrs hence . . . .  I'll try just restarting over from one of the other systems and see if that promotes the dysfunctional behaviors.  : - )

**Comment by geckolinux** [comment]
Thanks, let us know how it goes.
(Sorry for the slow reply to this issue, I replied several days ago via email but it somehow got lost.)

**Comment by esteelpaz** [comment]
@geckolinux 

No worries on it, I did "notice," as usually you are "all over it" as we say in SoCal . . . taking care of biznez and all that.  In this case I just had to reboot out of Gecko to solve the issue . . . .  The problem did "repeat" which is why I posted on it, but so far has not repeated again . . . .  We shall see.

**Comment by esteelpaz** [comment]
@geckolinux 

On this thread topic, I was using the Sys76 laptop edition of Gecko MATE rolling yesterday to do some writing, and even though I had "disable touchpad while typing" checked I noticed what I used to find in various LM distros on my Mac laptop, what was called then a "synaptics" problem . . . which I didn't have the chops to go into the synaptics file and re-write it, so the problem wasn't solved.

What happens is even holding the thumbs over the touchpad seems to move the cursor to another place, and/or yesterday it was "highlighting" sections of the text and I couldn't get it to un-highlight.

I use an external mouse, so possibly that causes "confusion" . . . I had to check "disable touchpad" to stop the problem, which "worked," but requires an additonal step, etc.  Don't know if that relates to the keyboard function issue, which didn't happen with this touchpad issue.

**Comment by esteelpaz** [comment]
@geckolinux 

I'm not sure that this topic is "closed," . . . I just haven't had time to get back to the laptop to try typing w/o ext keyboard . . . I possibly have ten linux distros spread across three computers, just don't have time to run them all.  Maybe tonight or Friday morning I'll be running the Gaz15 w/o ext keyboard, and I'll report back on it when I get a chance.

But, on a "lighter" note, posting here because I'm pressed for time . . . ran the "1561 package" upgrade in the Gecko Plasma install that was running nicely this morning . . . on reboot, once again back into the "grub menu is wiped" problem, and worse, it even went into the EFI disk and wiped that out as well . . . .  When I get the chance I'll have to boot into something with the Super Grub 2 disk and try to reinstall grub.  Something is still creating problems in the SUSE paradigm relating to grub menu for multi-boot conditions . . . .

**Comment by esteelpaz** [comment]
@geckolinux 

Yep, at work, fired up Gecko . . . tried to respond to a post and I have "disable while typing" selected, waved my thumbs over the touchpad and "zip" the typing box disappeared!!!  Problem remains, I believe in "synaptics"????

**Comment by geckolinux** [comment]
You should be able to remove `xf86-input-synaptics`, it's been replaced by libinput.

**Comment by esteelpaz** [comment]
@geckolinux 

OK, I managed to persuade zypper to remove that package . . . and I 'm waving my thumbs near the touchpad . . . seems so far I am able to get this typed out without having the cursor disappear or words highlighted and erased . . . .  

I'll "monitor" the situation for a tad bit . . . .  : - )))

[edit, a little bit later]:  So I rebooted the machine, and so far I've typed a couple of emails and haven't had the problem re-occur.  Pretty dramatic earlier, to have the IG message jump from one thread to another person's thread, open it, and start typing there.  Hopefully we are moving on to a fresh problem and leaving this one behind, etc.
